test/CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ if(SINGULARITY_USE_EOSPAC AND SINGULARITY_USE_HDF5 AND SINGULARITY_TEST_SESAME)
                              ${CMAKE_SOURCE_DIR}/utils/ports-of-call
                              ${CMAKE_SOURCE_DIR}/utils/sesame2spiner)
 endif()
-catch_discover_tests(eos_unit_tests)
+catch_discover_tests(eos_unit_tests PROPERTIES TIMEOUT 60)
```

Comment:
What's this timeout for?

Reply:
When I was dealing with undefined behavior, the loops would execute for 1500 seconds. While I could CTRL+C the tests, it didn't let the tests end to give me a clue about what was going on. Adding this timeout just makes it so each test has 60 seconds to complete instead of the default 1500.

At worst, this change does nothing at the moment and may need to be changed if somebody introduces a really long test. At best though, it will help somebody debug faster in the future if a test is misbehaving.

Reply:
ok seems fine for now.